src/sentry/utils/security/orgauthtoken_token.py

```diff
@@ -0,0 +1,48 @@
+from base64 import b64decode, b64encode
+from datetime import datetime
+from uuid import uuid4
+
+from django.conf import settings
+
+from sentry.utils import json
+
+SENTRY_ORG_AUTH_TOKEN_PREFIX = "sntrys_"
+
+
+def generate_token(org_slug: str, region_url: str):
+    sentry_url = settings.SENTRY_OPTIONS.get("system.url-prefix")
+    payload = {
+        "iat": datetime.utcnow().timestamp(),
+        "url": sentry_url,
+        "region_url": region_url,
+        "org": org_slug,
+    }
+    secret = b64encode(uuid4().bytes).decode("ascii").rstrip("=")
```

Comment:
This is generating a relatively short string and the `uuid4` library, while generally secure has no guarantees around generating the values securely.

I'd lengthen this and use the `secrets` library as it is cryptographically secure.

```suggestion
    secret = b64encode(secrets.token_urlsafe(nbytes=32).encode('utf-8'))
```